.idea/compiler.xml

```diff
@@ -7,6 +7,7 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="EkaterinaDubinina" />
+        <module name="EkaterinaDubinina_" />
       </profile>
     </annotationProcessing>
   </component>
```

Comment:
Why IDE related file in the commit?

Reply:
All IDE related files added to the gitignore.